docs/contributing/committing.rst

```diff
@@ -8,6 +8,9 @@ or for anyone interested in the process of getting code committed to Wagtail.
 Code should only be committed after it has been reviewed
 by at least one other reviewer or committer,
 unless the change is a small documentation change or fixing a typo.
+If additional code changes are made after the review, it is OK to commit them
+without further review if they are uncontroversial and small enough that no
+bugs are likely to be introduced.
```

Comment:
The word "likely" seems a bit weak here - it feels like there's too much room for people to say "I'm a really careful programmer - of course it's unlikely that I introduced new bugs" :-)

Maybe: "small enough that there is minimal chance of introducing new bugs"?